testsuite/MDAnalysisTests/utils/test_units.py

```diff
@@ -54,7 +54,7 @@ def test_unicode(self):
 
     def test_unicode_encoding_with_symbol(self):
         try:
-            assert_equal(units.lengthUnit_factor[u"Å"], 1.0)
+            assert_equal(units.lengthUnit_factor[u"Å"], 1.0)
```

Comment:
ok this is confusing.  I tried changing this to the Å my keyboard has and it can't commit as it doesn't see any changes...

Reply:
I just tried to reset the old utf character. Maybe it works....